pact.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       2.2
 name:                pact
-version:             4.1.3
+version:             4.2.0.1
```

Comment:
4.2.1? Or are we keepin the weed alive?

Reply:
this is a pre-release number (used in https://github.com/kadena-io/chainweb-node/pull/1341)